app/src/main/java/com/droidhats/campuscompass/viewmodels/SplashViewModel.kt

```diff
@@ -0,0 +1,25 @@
+package com.droidhats.campuscompass.viewmodels
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class SplashViewModel : ViewModel() {
+
+    //set as false to identify instantiations are not complete
+    private val mutableLiveData = MutableLiveData<Boolean>().apply {
+        value = false
+    }
+
+    //example of a value that can be accessed from SplashFragment
```

Comment:
What do you mean by example? Is this code necessary or not?

Reply:
It's not necessary, it was originally there to show how data would be passed between fragments but I forgot to update it. Good catch!